unitTests/utils/DomTest.ts

```diff
@@ -615,6 +647,103 @@ export function DomTests() {
         Simulate.removeJQuery();
       });
 
+      describe('with native set to true', () => {
+        beforeEach(() => {
+          Dom.native(true);
+        });
+
+        afterEach(() => {
+          Dom.native(false);
+        });
+
+        it('using "on" should work properly', () => {
+          const spy = jasmine.createSpy('spy');
+          new Dom(el).on('click', spy);
+          el.click();
+          expect(spy).toHaveBeenCalled();
+
+          const spy2 = jasmine.createSpy('spy2');
+          new Dom(el).on('foo', spy2);
+          new Dom(el).trigger('foo', {
+            detail: {
+              lorem: 'ipsum'
+            }
+          });
+
+          expect(spy2).toHaveBeenCalledWith(
+            jasmine.any(Event),
+            jasmine.objectContaining({
+              detail: {
+                lorem: 'ipsum'
+              }
+            })
+          );
+
+          const spy3 = jasmine.createSpy('spy3');
+          new Dom(el).on(['1', '2', '3'], spy3);
+          const events = ['1', '2', '3'].map(evt => {
+            return new CustomEvent(evt);
+          });
+          events.forEach(evt => {
+            el.dispatchEvent(evt);
+          });
+          expect(spy3).toHaveBeenCalledTimes(3);
+        });
+
+        it('using "on" with non alpha numeric character should work properly', () => {
+          const spy = jasmine.createSpy('spy');
+          new Dom(el).on('this contains space', spy);
+          new Dom(el).trigger('thiscontainsspace');
+          expect(spy).toHaveBeenCalledTimes(1);
+        });
+
+        it('using "one" should work properly', () => {
+          const spy = jasmine.createSpy('spy');
+          new Dom(el).one('click', spy);
+          el.click();
+          el.click();
+          el.click();
+          expect(spy).toHaveBeenCalled();
```

Comment:
Since we have an `expect` checking the number of times called, we could remove this one.